dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/internal/common/cdi/SpanProxy.java

```diff
@@ -27,6 +29,17 @@ public class SpanProxy implements Span {
 
     private static final TraceComponent tc = Tr.register(SpanProxy.class);
 
+    @Override
+    public Scope makeCurrent() {
+        //We can't do anything useful here but this will prevent the proxy itself becoming the current span.
+        return Span.current().makeCurrent();
```

Comment:
I think this change will work, but it would be simpler to return `Scope.noop()`

Reply:
Any risk that code which somehow ends up calling makeCurrent on our SpanProxy will expect the real current span from makeCurrent() and cause problems if it gets a noop?

Reply:
Should be ok because this method doesn't return the current span, it returns a `Scope` which can be used to stop it being current.

In this case we're not changing the current span (this object always represents the current span, so calling `makeCurrent` will never change anything), so returning a `Scope` that does nothing is fine.